### ISSUE TITLE
Add the ability to notify multiple Telegram Chat-IDs

### ISSUE THREAD

**Issue by nklsfrt** [Der-Henning/tgtg#71]
I think it'd be great if it was possible to have Telegram notifications sent to multiple Chat-IDs at once.

As far as I can tell all it would take is for the config parser to be able to handle a list of IDs supplied via `TELEGRAM_CHAT_ID=` and then iterating over the IDs when sending the notifications from `telegram.py`.

I'd make a PR myself, but I don't really get how the config parsing works, sorry.

**Comment by Der-Henning** [comment]
Hi @nklsfrt. Thank you for your suggestion.
Why would you like to use multiple chats? Just curious.
This feature should actually be very simple to implement and shouldn't have any impact on the current functionality.
I will consider it for the next release.

**Comment by nklsfrt** [comment]
> Why would you like to use multiple chats? Just curious.

My girlfriend and I would both like to be notified whenever one of our favorite places becomes available.

In general it would mean you don't have to run an instance per user that is to be notified, which makes the entire process a bit more flexible regardless of whatever your use-case might be.

Thank you for considering my suggestion!

**Comment by Dracula06** [comment]
Same thing, Me and my wife would both like notifications if possible. Thanks if you implemant.

**Comment by Der-Henning** [comment]
Added support for multiple chat ids in 1b0d8e6.
To send notifications to multiple chat ids, just add them as a comma separated list.
I will create a new Release v1.9.0 including this update until tomorrow. It is already available in the latest docker image.

**Comment by nklsfrt** [comment]
So I've tested the latest docker image, and it doesn't work as intended.

It seems to properly parse the IDs, since it does print each of them in the start up log, but instead of sending a message to each ID, it just sends as many messages to the first ID, as I have IDs specified.

I _think_ you need to set the `chat_id` inside `send_message()` to the var you use for iteration instead of `self.chat_id`? 
```
for ID in self.chat_id.split(','):
                try:
                    self.bot.send_message(
                        chat_id=ID,
                        text=message,
                        parse_mode=fmt,
                        timeout=60
                    )
                except Exception as err:
                    log.error(err)
                    #raise TelegramConfigurationError()
```
Or call it `self.chat_ids` so you can iterate over it using `chat_id` to avoid confusion?
I hope you got my point, my coding knowledge is limited at best.

**Comment by Der-Henning** [comment]
Yep. You are right. Fixed it abdc3ae.

**Comment by nklsfrt** [comment]
Alright, confirmed working.
Thank you, much appreciated!
